packages/picker/src/MobileController.ts

```diff
@@ -41,12 +57,8 @@ export class MobileController extends InteractionController {
             () => this.handlePointerdown(),
             { signal }
         );
-        this.target.addEventListener(
-            'focus',
-            (event: FocusEvent) => this.handleButtonFocus(event),
-            {
-                signal,
-            }
-        );
+        this.target.addEventListener('focusout', () => this.handleFocusOut(), {
```

Comment:
How are these events removed?

Reply:
All of the eventListeners that I'm adding have an argument `signal` which binds them to the abortController and in the `InteractionController.ts` we abort all those signals in the hostDisconnected method.